cellpack/autopack/DBRecipeHandler.py

```diff
@@ -209,14 +209,9 @@ def should_write(self, db, recipe_data):
         if db_docs and len(db_docs) >= 1:
             for doc in db_docs:
                 db_data = db.doc_to_dict(doc)
-                shallow_match = True
                 for item in CompositionDoc.SHALLOW_MATCH:
                     if db_data[item] != local_data[item]:
-                        print(db_data[item], local_data[item])
-                        shallow_match = False
                         break
```

Comment:
This break statement should probably be a return statement? It looks like it only breaks out of the inner loop right now.

```suggestion
                        return None, None
```

Reply:
yeah I know it's confusing, the readability of this function is truly a concern for us. Let's address it during our group programming session this week.